### PR TITLE
Add ScalaFX example

### PR DESCRIPTION
**Pull request by wsargent**
This is a documentation addition to show how to use ScalaFX with Ammonite.

It's taken from https://github.com/wsargent/scalafx-ammonite which is a worked example.

### CODE REVIEW

**Review by alexarchambault — APPROVED** [review]
Thanks!

File: readme/Cookbook.scalatex [hunk 1]
```diff
@@ -160,6 +160,71 @@
       res12: Int = 6
     @p
       This is a level of live interactivity which is traditionally hard to come by in the world of desktop GUI applications, but with the Ammonite REPL, it's totally seamless
+    @p 
+      You can also import JavaFX applications, using ScalaFX.  This is easiest if you create a file `jfx.sc` to contain the relevant imports:
+
+    @h1.scala
```
Comment (keynmol) [inline readme/Cookbook.scalatex:166]:
```suggestion
    @hl.scala
```

Comment (wsargent) [reply readme/Cookbook.scalatex:166]: derp

**Comment by wsargent** [comment]
Updated everything to have `h1` finally

**Comment by wsargent** [comment]
Can someone commit this?

**Comment by sake92** [comment]
@wsargent sorry for waiting, I can merge it. I tried it and it works good on Win10, JDK11.  
It would be great to have a possibility of calling `main()` interactively multiple times, since that's the whole point of using Ammonite I guess.. 😄   
First time runs fine, second time:
> @ main()
java.lang.IllegalStateException: Application launch must not be called more than once
  com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:175)
  com.sun.javafx.application.LauncherImpl.launchApplication(LauncherImpl.java:156)
  javafx.application.Application.launch(Application.java:227)
  scalafx.application.JFXApp3.main(JFXApp3.scala:294)
  scalafx.application.JFXApp3.main$(JFXApp3.scala:286)
  ammonite.$sess.cmd0$JFXApp3Demo$.main(cmd0.sc:19)
  ammonite.$sess.cmd0$.main(cmd0.sc:16)
  ammonite.$sess.cmd3$.<clinit>(cmd3.sc:1)

**Pull request merged**